client/my-sites/checkout/src/components/cost-overrides-list.tsx

```diff
@@ -246,17 +269,33 @@ function LineItemCostOverride( {
 	costOverride: LineItemCostOverrideForDisplay;
 	product: ResponseCartProduct;
 } ) {
+	const isPriceIncrease = doesIntroductoryOfferHavePriceIncrease( product );
+	if ( isPriceIncrease ) {
+		return (
+			<div className="cost-overrides-list-item" key={ costOverride.humanReadableReason }>
+				<LineItemIntroOfferCostOverrideDetail product={ product } />
+			</div>
+		);
+	}
 	return (
 		<div className="cost-overrides-list-item" key={ costOverride.humanReadableReason }>
-			<span className="cost-overrides-list-item__reason">{ costOverride.humanReadableReason }</span>
+			<span
+				className={
+					isPriceIncrease
```

Comment:
would `isPriceIncrease` be always `false` here since we would be returning early whenever it's `true` right above?

Reply:
Oh, hah. Good catch. I think this was left over from before I added that simplified early return.